test/run-e2e.sh

```diff
@@ -5,4 +5,4 @@ set -x
 
 readonly PKGDIR=sigs.k8s.io/gcp-compute-persistent-disk-csi-driver
 
-go test --timeout 30m --v "${PKGDIR}/test/e2e/tests" --run-in-prow=true --delete-instances=true --logtostderr $@
+go test --timeout 45m --v "${PKGDIR}/test/e2e/tests" --run-in-prow=true --delete-instances=true --logtostderr $@
```

Comment:
previously these tests were taking 27 min ish to run, and now they are taking 44 min. Why do these tests make the tests run so much longer?

Reply:
The changes from ControllerModifyVolume take 5-6 minutes to be reflected. I added tests for both IOPS/throughput, just IOPS, and just throughput, causing the tests to take longer

Reply:
Do these e2e tests not run in parallel?

Reply:
No, they don't run in parallel by default